modules/customize-certificates-understanding-default-router.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// authentication/certificates/replacing-default-ingress-certificate.adoc
+
+[id="understanding-default-ingress-{context}"]
+= Understanding the default ingress certificate
+
+By default {product-title} uses the Ingress Controller Operator to 
+create an internal CA and issue a wildcard certificate that is valid for 
+applications under the `.apps` sub-domain. This includes the web console 
+and CLI. 
+
+The internal infrastructure certificates are self-signed, 
+which might be perceived as bad practice by some security or PKI teams. 
```

Comment:
I've left this in its current place, but I've reworded the sentences so that the bad is immediately countered by why it isn't an issue in the same sentence.